include/circt/Dialect/Moore/MoorePasses.h

```diff
@@ -0,0 +1,37 @@
+//===- Passes.h - Moore pass entry points -----------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This header file defines prototypes that expose pass constructors.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CIRCT_DIALECT_MOORE_MOOREPASSES_H
+#define CIRCT_DIALECT_MOORE_MOOREPASSES_H
+
+#include "circt/Dialect/Moore/MooreOps.h"
+#include "circt/Support/Namespace.h"
+#include "mlir/Dialect/SCF/IR/SCF.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Support/LLVM.h"
+#include "llvm/ADT/ScopedHashTable.h"
+#include "llvm/ADT/TypeSwitch.h"
+#include <memory>
```

Comment:
You probably only need Moore ops and the pass.
```suggestion
#include "circt/Dialect/Moore/MooreOps.h"
#include "mlir/Pass/Pass.h"
#include <memory>
```